### PR TITLE
eval: added SyntaxError and ValueError handler

### PR DESCRIPTION
**Pull request by Adman**
* return error message when exception is raised

Signed-off-by: Adrian Matejov <a.matejov@centrum.sk>

### CODE REVIEW

**Comment by Adman** [comment]
cc @pepol @mrshu

File: basic.py [hunk 1]
```diff
@@ -85,4 +85,7 @@ def eval_(event):
     if len(input) < 1:
         return
 
-    return eval(input, {"__builtins__": None}, safe_dict)
+    try:
+        return eval(input, {"__builtins__": None}, safe_dict)
+    except (SyntaxError, ValueError) as e:
```
Comment (mrshu) [inline basic.py]: @Adman this could be a bit more general by just using `Exception` here.

**Comment by mrshu** [comment]
@Adman apart from that small comment, it looks good.

cc @JakubNvk

**Comment by Adman** [comment]
@mrshu Fixed. Please also remove branch eval-fix in this repository, thanks.

**Comment by mrshu** [comment]
@Adman cool, can you please also update the commit message? It doesn't really reflect what the code does now.

Otherwise LGTM!

**Comment by mrshu** [comment]
@Adman thanks, merging this!

**Pull request merged**